libs/auth/scripts/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-console": "off"
+  }
+}
```

Comment:
Pointing out that while this is technically relaxing `"no-console": "off"` from a per-file basis to the entire module, this seems fine because
1. We disabled `no-console` in every file anyway
2. It's a scripts module so logging is better practice anyway

Reply:
Yup, exactly my thinking here!